src/main/java/org/dataloader/DataLoaderOptions.java

```diff
@@ -171,16 +151,6 @@ public boolean batchingEnabled() {
         return batchingEnabled;
     }
 
-    /**
-     * Sets the option that determines whether batch loading is enabled.
-     *
-     * @param batchingEnabled {@code true} to enable batch loading, {@code false} otherwise
-     * @return a new data loader options instance for fluent coding
-     */
-    public DataLoaderOptions setBatchingEnabled(boolean batchingEnabled) {
-        return builder().setBatchingEnabled(batchingEnabled).build();
-    }
```

Comment:
All the object -> mutate -> new object methods are removed and now only in the builder